plenum/server/replica.py

```diff
@@ -452,6 +453,8 @@ def __init__(self, node: 'plenum.server.node.Node', instId: int,
 
         HookManager.__init__(self, ReplicaHooks.get_all_vals())
 
+        self.consensus_provider = ConsensusDataProvider(self.name)
```

Comment:
I would prefer `consensus_data_provider`. `consensus_provider ` is too generic.

Reply:
In new services I just use `_data`, indicating this is some shared data. I do agree this may look way too generic, especially in huge replica class, however full name looks quite verbose. What about naming it `_consensus_data` here?